modules/installation-aws-upi-installation.adoc

```diff
@@ -0,0 +1,187 @@
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: test-tkh7l
+      machine.openshift.io/cluster-api-machineset: test-tkh7l-worker-us-east-2a
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        machine.openshift.io/cluster-api-cluster: test-tkh7l
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: test-tkh7l-worker-us-east-2a
+    spec:
+      metadata:
+        creationTimestamp: null
+      providerSpec:
+        value:
+          ami:
+            id: ami-0eecbb884c8b35b1e <1>
+          apiVersion: awsproviderconfig.openshift.io/v1beta1
+          blockDevices:
+          - ebs:
+              iops: 0
+              volumeSize: 120
+              volumeType: gp2
+          credentialsSecret:
+            name: aws-cloud-credentials
+          deviceIndex: 0
+          iamInstanceProfile:
+            id: test-tkh7l-worker-profile
+          instanceType: m4.large
+          kind: AWSMachineProviderConfig
+          metadata:
+            creationTimestamp: null
+          placement:
+            availabilityZone: us-east-2a
+            region: us-east-2
+          publicIp: null
+          securityGroups: <2>
+          - filters:
+            - name: tag:Name
+              values:
+              - test-tkh7l-worker-sg
+          subnet: <3>
+            filters:
+            - name: tag:Name
+              values:
+              - test-tkh7l-private-us-east-2a
+          tags:
+          - name: kubernetes.io/cluster/test-tkh7l
+            value: owned
+          userDataSecret:
+            name: worker-user-data
+      versions:
+        kubelet: ""
+status:
+  fullyLabeledReplicas: 1
+  observedGeneration: 1
+  replicas: 1
+----
+<1> Specify the {op-system-first} AMI to use for your worker nodes.
+<2> Specify the name of the worker security group that you created.
+<3> Specify the subnet for the machines.
+
+.. Replace the `securityGroups` stanza with one that specifies the name of the
+worker security group that you created:
++
+----
+securityGroups:
+- filters:
+  - name: tag:Name
+    values:
+    - test-tkh7l-worker-sg
+----
+
+.. Replace the `subnet` stanza with one that specifies the name of the subnet
+to deploy the machines on:
++
+----
+subnet: <3>
+  filters:
+  - name: tag:Name
+    values:
+    - test-tkh7l-private-us-east-2a
+----
+
+. View the machines in the `openshift-machine-api` namespace and confirm that
+they are launching:
++
+----
+$ oc get machines --namespace openshift-machine-api
+NAME                                 INSTANCE              STATE     TYPE         REGION      ZONE         AGE
+test-tkh7l-worker-us-east-2a-hxlqn   i-0e7f3a52b2919471e   pending   m4.4xlarge   us-east-2   us-east-2a   3s
+----
+
+. Complete the cluster installation:
++
+----
+$ bin/openshift-install wait-for finish
```

Comment:
Per latest installer sub-command, should replace 'finish' with 'cluster-ready'
`
./openshift-install version
./openshift-install v4.1.0-201904101232-dirty
built from commit 741d4db20a2d673eb79cf6dbd076001c9cb65753
release image registry.svc.ci.openshift.org/ocp/release:4.0.0-0.nightly-2019-04-10-182914

 ./openshift-install wait-for --help
<--snip-->
Available Commands:
  bootstrap-complete Wait until cluster bootstrapping has completed
  cluster-ready      Wait until the cluster is ready
<--snip-->
`

Reply:
Thank you! I've updated the command.